ci-operator/step-registry/cucushift/installer/rehearse/vsphere/ipi/customized-resourcepool/provision/cucushift-installer-rehearse-vsphere-ipi-customized-resourcepool-provision-chain.yaml

```diff
@@ -0,0 +1,14 @@
+chain:
+  as: cucushift-installer-rehearse-vsphere-ipi-customized-resourcepool-provision
+  steps:
+  - chain: ipi-conf-vsphere
+  - ref: ipi-conf-vsphere-customized-resourcepool
+  - ref: ipi-conf-vsphere-folder
```

Comment:
Do you want to cover customized folder in this install workflow?

Reply:
no. forgot to delete the step. thanks for reminding.